asciidoc-release-notes/DC-release-notes

```diff
@@ -0,0 +1,17 @@
+## --------------------------------- 
+## Doc Config File for Release Notes
+## ProductName
+## ---------------------------------
+##
+## Basics
+MAIN=release-notes.adoc
+ADOC_POST=yes
+ADOC_TYPE=article
```

Comment:
In one of the other suggestions, you have suggested removing `attributes.adoc` from the MANIFEST file. Was that by mistake? Because if not, then this line is only in the DC file now.

Reply:
Having this in file should definitely suffice. I think the DC file might be the best option too.

The suggestion was also not a mistak, though the context may not have been clear enough. I suggested to remove `attributes.adoc`: The separation between "functional" AsciiDoc attributes (in attributes.adoc) and AsciiDoc attributes that we use as text macros/XML entities (entities-product and entities-generic) is completely artificial. To that end, it does not make any sense to me to ship a _three_ attribute files.

The idea behind having two attribute files is that one of them is generic and auto-updateable, the other one is specific to the product and should only ever be updated directly within the repo.

In any case, the attributes from `attributes.adoc` can be in either of the other two attributes files as well. There is nothing special about them. Hence, I'd just split up the values from `attributes.adoc` between the two other files, wherever they fit best.